### PR TITLE
set-up-and -mobile-version -skeleton 

### PR DESCRIPTION
**Pull request by letarulinda**
-created HTML and CSS files for the project.
-Modified the README.md 
-Added Social Media Icons to the project
-Added gitworkflows to the project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="index.css" rel="stylesheet" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>My Logo</header>
```
Comment (OmarRamoun) [inline index.html]: - [ ] [REQUIRED] Please, make sure to wrap the logo in a `<a>` tag as part of project requirments.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <h1>I’m Mariane Glad to see you!</h1>
+    <p>
+      I’m a software developer! I can help you build a product , feature or
+      website Look through some of my work and experience! If you like what you
+      see and have a project you need coded, don’t hestiate to contact me.
+    </p>
+    <p>LET’S CONNECT</p>
+  </body>
```
Comment (OmarRamoun) [inline index.html]: - [ ] [REQUIED] Please, add a section for the social media links as seen in the Figma Design

File: index.css [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.header {
+  color: blue;
+}
```
Comment (OmarRamoun) [inline index.css]: - [ ] [REQUIRED] Please, add styles to the `index.css` file as requested in project requirements. Currently it has no styles

**Review by OmarRamoun — CHANGES_REQUESTED** [review]
# 🟡 Almost There! 🏃

## Hi @letarulinda 👋

## Welcome to Microverse! 🌎

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃

## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Especially if that's is your first project 1️⃣. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better job** 💼.

_I am going to guide you on how to fix them down below. 🤓_


## Highlights 🌟

- ✔️ descriptive commit messages 
- ✔️ descriptive PR title and description
- ✔️ no linter errors

- The code is top-notch! 💯 and have a huge potential.

## Required Changes ♻️

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — COMMENTED** [review]
Hi @letarulinda,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
You have submitted a project, that has linters errors.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## 
_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+      <br />
+      <div>
+        <h2 class="social media links">
+          <img src="assets/twittericon.svg" alt="bird-like-picture" />
+          <img src="assets/linkediniicon.svg" alt="recruiter website " />
+          <img src="assets/linkedinicon.svg" alt="recruiter website" />
+          <img src="assets/micon.svg" alt="letter-m-shaped word" />
+          <img src="assets/githubicon.svg" alt="cat-like-picture" />
+          <img src="assets/peaceicon.svg" alt="two-raised-figure" />
+        </h2>
```
Comment (Ismailco) [inline index.html]: - [ ] good work implementing the social links however you need to group them in a ``` ul``` and wrap every logo in a ```li``` tag, every logo should be a link so also kindly consider wrapping every ```img``` in a ```a``` tag so it's a link

File: index.html [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+      through some of my work and<br />experience! If you like what you see
+      and<br />
+      have a project you need coded, don’t<br />hestiate to contact me.
+    </p>
+    <div>
+      <br />
+      <p class="suggegiion-comment">LET’S CONNECT</p>
+      <br />
+      <div>
+        <h2 class="social media links">
```
Comment (Ismailco) [inline index.html]: - [ ] [OPTIONAL] Please don't use ```hx``` to wrap element on your page, this is for the heading

File: index.html [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <a class="logo"> My Logo</a>
+    <img src="assets/Union.jpg" alt="menu-bar" />
```
Comment (Ismailco) [inline index.html]: - [ ] [OPTIONAL] Please consider wraping the header section (nav bar/top bar) in a ```header``` tag, this is a requirement

Comment (Ismailco) [inline index.html]: - [ ] [OPTIONAL] The hamburger button should have a ```nav``` tag, this is a requirement kindly implement it

File: index.html [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+    <section class="intro-wrapper">
```
Comment (Ismailco) [inline index.html:38]: - [ ] [OPTIONAL] Kindly wrap the whole Headline section in ``` section``` tag not just the header (```h1```)

File: index.css [hunk 8]
```diff
@@ -0,0 +1,20 @@
+body {
```
Comment (Ismailco) [inline index.css:1]: - [ ] Kindly consider styling your page according to the Figma design including the colors, margin/padding and align-items using flexbox. so your design is completed 

|Your design|Figma design|
|-|-|
|Screenshot_20220528_171403|Screenshot_20220528_171759|

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @letarulinda,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
Hi @letarulinda, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**